scripts/bump_extension_version.js

```diff
@@ -0,0 +1,153 @@
+const fs = require('fs')
+const path = require('path')
+const semVer = require('semver')
+
+function isMinorRelease(
+  prismaVersion,
+) {
+  const tokens = prismaVersion.split('.')
+  if (tokens.length !== 3) {
+    throw new Error(
+      `Version ${prismaVersion} must have 3 tokens separated by "." character.`
+    )
+  }
+  return tokens[2] === '0'
+}
+
+function currentExtensionVersion({
+  branch_channel
+}) {
+  switch (branch_channel) {
+    case 'master':
+    case 'dev':
+      return readFile({ fileName: "extension_insider" })
+    case 'latest':
+      return readFile({ fileName: "extension_stable" })
+    case 'patch-dev':
+      return readFile({ fileName: "extension_patch" })
+    default:
+      if (branch_channel.endsWith('.x')) {
+        return readFile({ fileName: "extension_patch" })
+      }
+  }
```

Comment:
Probably throw with an error in the else block? It is an unhandled case otherwise.